position_tolerance: 0.02 # meters, if elevator position is +/- than the server set success
cube:
  intake: 1.00
  low_node: 2.05
  middle_node: 3.00
  high_node: 4.00

vertical_cone:
  intake: 0.20
  low_node: 2.00
  middle_node: 3.00
  high_node: 4.10

base_towards_us_cone:
  intake: 1.00
  low_node: 2.00
  middle_node: 3.00
  high_node: 4.00

base_away_us_cone:
  intake: 1.00
  low_node: 2.00
  middle_node: 3.00
  high_node: 4.00

saftey_high: 2.00 # distance when fourbar must be at least a certain amount extended up high
saftey_intake: 0.5 # when below this we must be out a certain distance
